pom.xml

```diff
@@ -269,7 +269,6 @@ file comparators, endian transformation classes, and much more.
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
       <version>3.11</version>
-      <scope>test</scope>
```

Comment:
-1 we don't want this runtime dependency.

Reply:
thanks, remove runtime dependency